frontend/src/data/variables/AcsHealthInsuranceProvider.ts

```diff
@@ -16,7 +16,7 @@ class AcsHealthInsuranceProvider extends VariableProvider {
   }
 
   getDatasetId(breakdowns: Breakdowns): string {
-    if (breakdowns.hasOnlySex()) {
+    if (breakdowns.hasOnlySex() || breakdowns.hasOnlyAge()) {
       return breakdowns.geography === "county"
         ? "acs_health_insurance-health_insurance_by_sex_county"
```

Comment:
It's a bit confusing that the name for this dataset does not include age but we are using it to get age - can you either update the names of these "by_sex" datasets to include age or at minimum add a comment about this?

Reply:
Changed the dataset name